beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/model/HousekeepingMetadata.java

```diff
@@ -187,19 +201,29 @@ public Duration getCleanupDelay() {
 
   public void setCleanupDelay(Duration cleanupDelay) {
     this.cleanupDelay = cleanupDelay;
-    cleanupTimestamp = creationTimestamp.plus(cleanupDelay);
+    this.cleanupTimestamp = this.creationTimestamp.plus(cleanupDelay);
```

Comment:
Minor, but you don't need the `this` here as it's not ambiguous what you're referring to unlike `cleanupDelay` which in this case is both a method parameter and a member variable.